changelog/_unreleased/2020-09-17-directly-open-product-on-search-by-product-number.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Directly open product on search if there is only one result
+issue: NEXT-10857
+author: Stefan Wild
+author_email: stefan.wild@sewisoft.de 
+author_github: @wildi1
+---
+# Storefront
+Changed `\Shopware\Storefront\Controller\SearchController.php` to redirect direct to the detail page if there is only one result
```

Comment:
```suggestion
* Changed `\Shopware\Storefront\Controller\SearchController.php` to redirect direct to the detail page if there is only one result
```

Reply:
Done

Reply:
But that should be recognized by php psh.phar fix-cs too ;-)

Reply:
That's not a codestyle fix though :-P